led_strip/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.4.0
+
+- Support configurable SPI mode to contorl leds
+  - The interface still needs to be modified, the current version is only for technical verification
```

Comment:
what do you mean by "needs to be modified"? which public function or structure do you see needs a modification in the sooner future?